packages/devtools-local-toolbox/src/clients/chrome.js

```diff
@@ -38,51 +40,70 @@ function connectClient() {
   const deferred = defer();
 
   if(!getValue("chrome.debug")) {
-    return Promise.resolve(createTabs([]))
+    return Promise.resolve(createTabs([], {}))
   }
 
-  const webSocketPort = getValue("chrome.webSocketPort");
-  const url = `http://localhost:${webSocketPort}/json/list`;
-  networkRequest(url).then(res => {
-    deferred.resolve(createTabs(JSON.parse(res.content)))
-  }).catch(() => deferred.reject());
+  const port = getValue("chrome.port");
+  function fetch(http, options, callback) {
+    const {host, path, port} = options;
+    const url = `http://${host}:${port}${path}`;
+    return networkRequest(url).then(res => callback(null, res.content))
+  }
 
-  return deferred.promise;
+  return CDP.List({ port, fetch })
+    .then(tabs => createTabs(tabs, { browser: "chrome", type: "page" }))
+    .catch(err => {});
+}
+
+
+function connectNodeClient() {
+
+  if(true){
+    return Promise.resolve(createTabs([], {}))
+  }
```

Comment:
disabled for now while I sort out chrome. It will be enabled before this lands ofcourse